web/src/features/fireWeather/components/graphs/PercipGraph.tsx

```diff
@@ -0,0 +1,153 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
```

Comment:
File should be called PrecipGraph, not Percip - or - might as well just say PrecipitationGraph - no need to abbreviate?

Reply:
PrecipitationGraph it is!